deploy/bootstrap/.env

```diff
@@ -2,3 +2,4 @@ LOG_LEVEL=debug
 GOSSIP_PEERS=ws://us-east-1.global.nodes.staging.orbs-test.com:60001,ws://eu-central-1.global.nodes.staging.orbs-test.com:60001,ws://ap-northeast-1.global.nodes.staging.orbs-test.com:60001,ws://ap-northeast-2.global.nodes.staging.orbs-test.com:60001,ws://ap-southeast-2.global.nodes.staging.orbs-test.com:60001,ws://ca-central-1.global.nodes.staging.orbs-test.com:60001
 E2E_PUBLIC_API_ENDPOINT=0.0.0.0:51151
 E2E_NO_DEPLOY=true
+SMART_CONTRACTS_TO_LOAD=[] # [{address: "foobar", filename: "foobar-smart-contract"}]
```

Comment:
This should be set only for non-production environments. I didn't understand by this env filename what setup it's used for

Reply:
Let's put a comment that this is just a sample of how the .env file can look like, or even comment out the explicit staging settings and make it fail by default (this will force users to explicitly uncomment and set it)